scripts/leapfrog/ubuntu14-leapfrog.sh

```diff
@@ -98,11 +98,11 @@ pushd ${LEAPFROG_DIR}
     # Get the OSA LEAPFROG
     if [[ ! -d "openstack-ansible-ops" ]]; then
         git clone ${OA_OPS_REPO}
-        pushd openstack-ansible-ops
-            git checkout ${OA_OPS_REPO_BRANCH}
-        popd
         log "clone" "ok"
     fi
+    pushd openstack-ansible-ops
```

Comment:
You should add a git fetch --all here to ensure that you get the most up to date refs, otherwise the checkout may fail if it was done prior to the SHA you want to use existing.